test/benchmarks/src/main/scala/scala/collection/mutable/ArrayOpsBenchmark.scala

```diff
@@ -23,15 +23,43 @@ class ArrayOpsBenchmark {
   var strings: List[String] = _
   var integersA: Array[Int] = _
   var stringsA: Array[String] = _
-
+  var intIntA: Array[Array[Int]] = _
```

Comment:
I would have called this `intsAA` to match the other names better, but it's not important.